### PR TITLE
[OTA-2800] Handle different http status codes in DDV

### PR DESCRIPTION
**Pull request by piotrsmol**
Signed-off-by: Piotr Smol <piotrsmol.pvt@gmail.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/constants/index.js [hunk 1]
```diff
@@ -33,6 +33,16 @@ export const NO_VERSION_INFO = 'No information / Any';
...
 export const ARTIFICIAL = 'artificial';
 
+/*
+ *  Device
+ */
+
+export const DEVICE_STATUSES = {
+    UP_TO_DATE: 'UpToDate',
+    OUTDATED: 'Outdated',
+    ERROR: 'Error'
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/constants/index.js]: Keep it alphabetically please.

File: ota-plus-web/app/reactapp/src/helpers/deviceHelper.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      case HTTP_CODE_503_SERVICE_UNAVAILABLE:
+      case HTTP_CODE_504_GATEWAY_TIMEOUT:
+        message = 'Please try again later'
+        break;
+      default:
+        message = 'Never seen online'
+        break;
+    }
+  return message;  
+};
```
Comment (piotrcichonmee) [inline ota-plus-web/app/reactapp/src/helpers/deviceHelper.js]: Add new line at the end of file

File: ota-plus-web/app/reactapp/src/constants/httpCodes.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
 
 export const HTTP_CODE_200_OK = 200;
 export const HTTP_CODE_404_NOT_FOUND = 404;
+export const HTTP_CODE_503_SERVICE_UNAVAILABLE = 503;
+export const HTTP_CODE_504_GATEWAY_TIMEOUT = 504;
```
[the remark this answers is not in the document]
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/constants/httpCodes.js]: done

File: ota-plus-web/app/reactapp/src/constants/index.js (hunk 1 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/constants/index.js]: done

File: ota-plus-web/app/reactapp/src/helpers/deviceHelper.js (hunk 2 above)
Comment (piotrsmol) [reply ota-plus-web/app/reactapp/src/helpers/deviceHelper.js]: done

**Pull request merged**